upgrade/helpers/tasks.py

```diff
@@ -153,6 +155,41 @@ def sync_capsule_repos_to_upgrade(capsules):
     except KeyError:
         # If latest capsule repo is not created already(Fresh Upgrade),
         # So create new....
+        if to_version in ['6.4']:
```

Comment:
Move this as a separate function, in tasks module.

Reply:
Would need to refactor the whole sync_capsule_repos , as per release version
Should we do it ?